volatility3/framework/symbols/windows/extensions/mft.py

```diff
@@ -2,23 +2,152 @@
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
 #
 
-from typing import Optional
+import logging
+from typing import Dict, Iterator, List, Optional, Tuple
 
-from volatility3.framework import objects, constants, exceptions
+from volatility3.framework import constants, exceptions, interfaces, objects
+
+vollog = logging.getLogger(__name__)
 
 
 class MFTEntry(objects.StructType):
     """This represents the base MFT Record"""
 
-    def get_signature(self) -> str:
+    def __init__(
```

Comment:
I'm not sure I'm keen on this, because it then means anyone following a pointer to an MFTEntry won't actually be able to use it because it won't have a symbol_table_name (even though it will have been pulled from, presumably, the right symbol table)?